src/app/app.component.html

```diff
@@ -1,2 +1,4 @@
 <router-outlet></router-outlet>
-<app-main-content></app-main-content>
+<div class="main">
```

Comment:
Why not using a `main` html component directly?

Reply:
to avoid the pre-defined styles in angular

Reply:
Angular adds predefined styles? O.o